codalab/rest/interpret.py

```diff
@@ -245,6 +246,14 @@ def fetch_interpreted_worksheet(uuid):
     # Fetch items.
     worksheet_info['source'] = get_worksheet_lines(worksheet_info)
 
+    for item in worksheet_info['items']:
+        if item[-1] is None:
+            lines = worksheet_info['source']
+            new_items = parse_worksheet_form(lines, local.model, request.user, uuid)
+            worksheet_info = get_worksheet_info(uuid, fetch_items=True)
+            update_worksheet_items(worksheet_info, new_items)
```

Comment:
Rather than re-saving the worksheet, can we just assign it an arbitrary sort key that is at the bottom of the worksheet? Or would that cause a problem?

Reply:
yeah that should work too

Reply:
updated, let me know what you think

Reply:
@epicfaace can I get a review on the new approach? thanks!